.travis.yml

```diff
@@ -1,3 +1,8 @@
+dist: trusty
 sudo: false
 language: ruby
+rvm:
+  - 1.9.3
+  - 2.4.0
+  - jruby-19mode
```

Comment:
I think this is not JRuby 1.7.x anymore. You might want to add `jruby-1.7.19` and `jruby-9.1.7.0` or something like that.